### PR TITLE
Added a homework2

### PR DESCRIPTION
**Pull request by sasha-si**

### CODE REVIEW

File: homework/siuo/fizzBuzz.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const count = 100;
+
+for (let i = 1; i <= count; i++) {
+   if (i % 5 === 0 && i % 3 === 0) {
+      console.log('FizzBuzz');
+   } else if (i % 5 === 0 && i % 3 !== 0) {
```
Comment (kovm) [inline homework/siuo/fizzBuzz.js]: `&& i % 3 !== 0` is redundant here

File: homework/siuo/index.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
-function bubbleSortConcept1(arr) {
-
-    for (let j = arr.length - 1; j > 0; j--) {
-      for (let i = 0; i < j; i++) {
-        if (arr[i] > arr[i + 1]) {
-          let temp = arr[i];
-          arr[i] = arr[i + 1];
-          arr[i + 1] = temp;
-        }
+function isPolindrome (str) {
```
Comment (kovm) [inline homework/siuo/index.js]: isPolindrome() throws an exception
isPolindrome(null) throws an exception
isPolindrome([1,2,1]) return true

File: homework/siuo/index.js [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
+   const someStr = str.toString();
+   let i = 0;
+   let j = someStr.length - 1;
+   let yesOrNo = true;
+   for (let l = 0; l <= someStr.length / 2; l++) {
+      if (someStr[i] !== someStr[j]) {
+         yesOrNo = false;
```
Comment (kovm) [inline homework/siuo/index.js]: if argument is not a palindrome immediately return no need to check the rest of the line

**Comment by sasha-si** [comment]
I made changes

**Pull request merged**